ground/gcs/src/plugins/coreplugin/coreconstants.h

```diff
@@ -226,7 +226,7 @@ const char * const ICON_PLUGIN           = ":/core/images/pluginicon.png";
 const char * const ICON_EXIT             = ":/core/images/exiticon.png";
 const char * const ICON_OPTIONS          = ":/core/images/optionsicon.png";
 const char * const ICON_HELP             = ":/core/images/helpicon.png";
-const char * const ICON_GCS              = ":/core/gcs_logo_128";
+const char * const ICON_GCS              = ":/core/gcs_nontrans_64";

```

Comment:
every other thing is setting the title bar icon to 32pix; (maybe i should change those to 64).  I don't have art for 128pix